lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -347,12 +347,11 @@ void addFunctionPasses(SILPassPipelinePlan &P,
   P.addDevirtualizer();
   P.addARCSequenceOpts();
 
-  if (!P.getOptions().SerializeStdlibWithOwnershipWithOpts) {
+  if (P.getOptions().EnableOptimizedOSSASerialization) {
     // We earlier eliminated ownership if we are not compiling the stdlib. Now
     // handle the stdlib functions, re-simplifying, eliminating ARC as we do.
     P.addCopyPropagation();
     P.addSemanticARCOpts();
-    P.addNonTransparentFunctionOwnershipModelEliminator();
```

Comment:
This needed to go since otherwise, we would lower ownership too early. Luckily since I thinkoed a few lines before, this didn't impact the stdlib and I was able to merge.